### PR TITLE
fixed animation tangent transform

### PR DESCRIPTION
**Pull request by UX3D-becher**
animation tangents have been transformed wrongly. This commit fixes the issue, by applying transformation on control points instead of gradients

### CODE REVIEW

File: addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animation_sampler_keyframes.py [hunk 1]
```diff
@@ -197,10 +201,10 @@ def all_equal(lst):
                                      "Baking animation because of differently located keyframes in one channel")
         return True
 
-    # Baking is required when the animation targets a quaternion with bezier interpolation
-    if channels[0].data_path == "rotation_quaternion" and interpolation == "BEZIER":
-        gltf2_io_debug.print_console("WARNING",
-                                     "Baking animation because targeting a quaternion with bezier interpolation")
-        return True
+    # # Baking is required when the animation targets a quaternion with bezier interpolation
+    # if channels[0].data_path == "rotation_quaternion" and interpolation == "BEZIER":
```
Comment (emackey) [inline addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animation_sampler_keyframes.py]: Is this comment and code still needed?  Future readers will be confused by the comment saying something is required, followed by code to do it being commented out.

If it's really not needed anymore, let's remove it.

Comment (UX3D-becher) [reply addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animation_sampler_keyframes.py]: forgot about that, thanks!
I'll add the removal to the "prune unused code" pull request

Comment (emackey) [reply addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animation_sampler_keyframes.py]: Thanks!  #397

**Comment by julienduroure** [comment]
Seems #205 still has a tangent issue.
Keyframe of scale is OK, but handle seems to be KO

**Pull request merged**